homeassistant/components/elkm1/__init__.py

```diff
@@ -101,9 +101,11 @@ def hostname_from_url(url: str) -> str:
 
 def _host_validator(config: dict[str, str]) -> dict[str, str]:
     """Validate that a host is properly configured."""
-    if config[CONF_HOST].startswith("elks://"):
+    if config[CONF_HOST].startswith(("elks://", "elksv1_2://")):
```

Comment:
tweaked it here since otherwise I felt bad leaving yet another comment on a 2 line PR

Reply:
Appreciated. I didn’t remember that you could do that.